build/builders.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Using Docker Build
+description: About building with existing builders
+keywords: build, buildx, buildkit
+---
+
+
+## Docker Build Components
+
+Currently Engine holds two build components:
+* Legacy Builder - Old builder implementation. It's executed through the `docker build` command.
+* Buildkit - Active by default in new and latest installations of Docker. It's executed through the `docker buildx build` command.
+
+### Docker Build Default Configuration
+
+The backend you have active depends on your Docker's installation.
+If you have installed:
+
+* __Docker Desktop (all platforms)__: BuildKit is enabled by default; no additional configuration is required.
+* In Linux, if you have:
+  * Engine 18.09 and higher you have BuildKit enabled by default.
+  * Engine releases older than 18.09 only include the Legacy Builder component, so we highly recommend you upgrade your installation or alternatively install Docker Desktop.
+
+
+### Docker Build Configuration
+
+Backend configuration is performed using the `DOCKER_BUILDKIT` environment variable. 
+
+If not yet set as default, to set the BuildKit as the default backend behind the `docker build` command, you can:
+
+__Method 1__
+
+From your console, run:
+
+```console
+$ DOCKER_BUILDKIT=1 docker build .
+```
+` 
+Depending on the value for this variable:
+* __DOCKER_BUILDKIT=1__ - BuildKit is executed both when you issue the `docker build` and `docker buildx build` command. With this configuration `docker build` becomes an alias for `docker buildx build`.
+* __DOCKER_BUILDKIT=0__ - When you issue `docker build` the build is executed using the Legacy Builder and BuildKit when you issue the `docker buildx build` command.
```

Comment:
This is confusing and I don't think we need that level of explanation as it's already implied previously.

___

> * __DOCKER_BUILDKIT=1__ - BuildKit is executed both when you issue the `docker build` and `docker buildx build` command.

Also this is not correct, BuildKit is always enabled with buildx and does not require `DOCKER_BUILDKIT=1` environment variable so we should only talk about `docker build` command here.

> With this configuration `docker build` becomes an alias for `docker buildx build`.

This is correct for Docker 22.06 but again this is confusing as user might think it would work with Docker 20.10 which is not the case.

> When you issue `docker build` the build is executed using the Legacy Builder and BuildKit when you issue the `docker buildx build` command.

Remove " and BuildKit when you issue the `docker buildx build` command."

___

If you still want to add these lines, then 

```suggestion
Depending on the value for this variable:
* __DOCKER_BUILDKIT=1__ - BuildKit is enabled for your build and starting Docker 22.06, `docker build` becomes an alias for `docker buildx build`.
* __DOCKER_BUILDKIT=0__ - Your build will use the Legacy builder.
```